tendrl/gluster_integration/objects/volume/atoms/check_volume_available/__init__.py

```diff
@@ -0,0 +1,52 @@
+import etcd
+import gevent
+import subprocess
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.gluster_integration.objects.volume import Volume
+
+
+class CheckVolumeAvailable(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckVolumeAvailable, self).__init__(*args, **kwargs)
+
+    def run(self):
+        retry_count = 0
+        while True:
+            try:
+                volumes = NS._int.client.read(
```

Comment:
As discussed earlier, while volume creation we have volume name only available. This post run atom is for create volume flow. To load a specific volume we need volume uuid which is not available here.